src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -757,13 +757,26 @@ export class BrowsingContextImpl {
       );
     }
 
-    const result = await this.#cdpTarget.cdpClient.sendCommand(
-      'Page.captureScreenshot',
-      {clip: {...rect, scale: 1.0}}
-    );
-    return {
-      data: result.data,
-    };
+    try {
+      await this.#cdpTarget.cdpClient.sendCommand(
+        'Emulation.setDefaultBackgroundColorOverride',
+        {
+          color: {r: 0, g: 0, b: 0, a: 0},
+        }
+      );
+      return await this.#cdpTarget.cdpClient.sendCommand(
+        'Page.captureScreenshot',
+        {clip: {...rect, scale: 1.0}}
+      );
+    } finally {
+      await this.#cdpTarget.cdpClient
+        .sendCommand('Emulation.setDefaultBackgroundColorOverride', {
+          color: undefined,
+        })
+        .catch(() => {
+          void 0;
```

Comment:
Log some error within the catch block. It helps with debugging should this situation ever happen.